app/indexers/concerns/hyku_indexing.rb

```diff
@@ -22,7 +22,7 @@ module HykuIndexing
         solr_doc['valkyrie_bsi'] = object.kind_of?(Valkyrie::Resource)
         solr_doc['member_ids_ssim'] = object.member_ids.map(&:id) if object.kind_of?(Valkyrie::Resource)
         # TODO: Reinstate once valkyrie fileset work is complete - https://github.com/scientist-softserv/hykuup_knapsack/issues/34
```

Comment:
Is this comment now obsolete? Wondering if we should remove it?

Reply:
good catch!